primitives/consensus/babe/src/lib.rs

```diff
@@ -364,25 +364,25 @@ sp_api::decl_runtime_apis! {
 		#[changed_in(2)]
 		fn configuration() -> BabeGenesisConfigurationV1;
 
-		/// Returns the slot that started the current epoch.
-		fn current_epoch_start() -> Slot;
+		/// Returns the slot that started the current session.
+		fn current_session_start() -> Slot;
 
-		/// Returns information regarding the current epoch.
-		fn current_epoch() -> Epoch;
+		/// Returns information regarding the current session.
+		fn current_session() -> Session;
 
-		/// Returns information regarding the next epoch (which was already
+		/// Returns information regarding the next session (which was already
 		/// previously announced).
-		fn next_epoch() -> Epoch;
```

Comment:
This is a breaking change, we should keep the old names around at least temporarily.

Reply:
Hello @andresilva thanks a lot for the review, I'll do the needful. However, can you please elaborate on the breaking changes, so that I am clear on what areas of the code base to avoid making these changes to ? Since I am to go about this very selectively I'd very much not like to break anything in the process. :)

Reply:
In this case you should bump the `api_version` in the attribute above, and then add `renamed` attributes to all the methods here with the old names. Similar to what is done here: https://github.com/paritytech/substrate/blob/master/primitives/block-builder/src/lib.rs#L42